templates/taxi/car_list.html

```diff
@@ -2,16 +2,19 @@
 
 {% block content %}
   <h1>Car list</h1>
-  {% if car_list %}
+  
+  <a href="{% url 'taxi:car-create' %}" class="btn btn-primary mb-2">Create New Car</a>
+  
+  {% if object_list %}
     <ul>
-      {% for car in car_list %}
+      {% for car in object_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
+          <a href="{% url 'taxi:car-detail' pk=car.id %}">{{ car.model }}</a>
```

Comment:
Ensure that the links to car details are functioning correctly and that the user has the necessary permissions to view car details.